Adds jenkins agent vpn terraform scripts (#234)
terraform-google-modules/terraform-example-foundation: 0-bootstrap/README-Jenkins.md

1. Commit changes with `git add backend.tf` and `git commit -m 'Your message - Terraform Backend configuration using GCS'`
1. Push my-0-bootstrap branch to your repository YOUR_NEW_REPO-0-bootstrap with `git push`

### III. Configure VPN connection

Here you will configure a VPN Network tunnel to enable connectivity between the `prj-cicd` project and your on-prem environment. Learn more about [a VPN tunnel in GCP](https://cloud.google.com/network-connectivity/docs/vpn/how-to).
- Required information:
    - On-prem VPN public IP Address
    - Jenkins Master’s network CIDR (the example code uses "10.1.0.0/24")
    - Jenkins Agent network CIDR (the example code uses "172.16.1.0/24")
    - VPN PSK (pre-shared secret key)

> 1. Check in the `prj-cicd` project which are the gateway static IP address that has been reserved for your VPN. You **NEED** to inform your Network administrator those IP address so they configure the on-prem side of the VPN tunnels.

Reviewer: ```suggestion
1. Check in the `prj-cicd` project for the VPN gateway static IP addresses which have been reserved. These addresses are required by the Network Administrator for the configuration of the on-prem side of the VPN tunnels to GCP.
```
